analysis/priestdiscipline/src/modules/spells/Evangelism.tsx

```diff
@@ -11,6 +11,7 @@ import isAtonement from '../core/isAtonement';
 import Atonement from './Atonement';
 
 const EVANGELISM_DURATION = 6;
+const EVANGELISM_DURATION_MS = 6000;
```

Comment:
```suggestion
const EVANGELISM_DURATION_MS = EVANGELISM_DURATION * 1000;
```

having a second constant here is okay, but it shouldn't need updating separately.